src/sys/wait.rs

```diff
@@ -340,7 +340,7 @@ pub enum Id<'fd> {
     /// If the PID is zero, the caller's process group is used since Linux 5.4.
     PGid(Pid),
     /// Wait for the child referred to by the given PID file descriptor
-    #[cfg(linux_android)]
```

Comment:
`linux_android` means `any(target_os = "linux", target_os = "android")`, so you should be able to use this flag on Linux, [here](https://docs.rs/nix/latest/nix/sys/wait/enum.Id.html#variant.PIDFd) is the doc of this flag on Linux/glibc.

FYI, `linux_android` is defined [here](https://github.com/nix-rust/nix/blob/9c0bdaca98db0401545e00bd3fc8c6dcb65df557/build.rs#L24).

Reply:
Wow, sorry! I was fairly confused, thanks once again @SteveLauC.